index.js

```diff
@@ -102,9 +103,8 @@ ModernizrPlugin.prototype.validatePlugin = function (plugin) {
 ModernizrPlugin.prototype.apply = function (compiler) {
   var self = this;
 
-  (compiler.hooks ? compiler.hooks.afterCompile.tapAsync.bind(compiler.hooks.afterCompile, 'ModernizrWebpackPlugin') : compiler.plugin.bind(compiler, 'after-compile'))((compilation, cb) => {
-
-    var buildOptions = assign({}, self.options);
+  compiler.hooks.afterCompile.tapAsync('ModernizrWebpackPlugin', (compilation, cb) => {
```

Comment:
What was the reason for skipping the `compiler.hooks` check here (and below)?

Reply:
Since the package has been upgraded to use the Webpack 5, if you try using `compiler.plugin()` you'll get the message`DeprecationWarning: Tapable.plugin is deprecated. Use new API on .hooks instead`. 

So, I removed the checks and we can use the hooks APIs instead (https://webpack.js.org/api/compiler-hooks/).

I think that's the official mention in the changelog: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#tapable-upgrade